wpimath/src/test/java/edu/wpi/first/math/kinematics/ChassisSpeedsTest.java

```diff
@@ -44,6 +46,38 @@ void testDiscretize() {
                 kEpsilon));
   }
 
+  @Test
+  void testDiscretizeMeasures() {
```

Comment:
Units-wise, what is this even testing? The test uses the input values as the reference, so I don't think this checks anything `testDiscretize()` doesn't.

Reply:
It looks like it just tests the ChassisSpeeds measure constructor, which should be the responsibility of a different test.

Reply:
This was mostly to test the functionality is the same given measures as inputs instead of raw doubles. It could be rewritten to test that the fields are identical, and let the existing discretization test cover that functionality